app/controllers/issues/evidence_controller.rb

```diff
@@ -0,0 +1,19 @@
+class Issues::EvidenceController < IssuesController
+  before_action :set_issue, only: [:index]
+  before_action :set_affected_nodes, only: [:index]
+
+  EXTRA_COLUMNS = ['Created by', 'Created', 'Updated'].freeze
+  SKIP_COLUMNS = ['Title', 'Label'].freeze
+
+  def index
+    @evidence_columns = ['Node'] | (collection_field_names(@issue.evidence) - SKIP_COLUMNS) | EXTRA_COLUMNS
```

Comment:
Not a fan of `SKIP_COLUMNS`, Title and Label are "pseudofields" that come from:

https://github.com/dradis/dradis-ce/blob/develop/app/models/evidence.rb#L33

For all intents and purposes, they're real fields, so I don't think we need to skip them. I'd definitely hide them by default, but I don't think we should obliterate them.

Also, Label and Node are prolly equivalent?

Reply:
I think we can ditch the skip addition and unify this like the rest.

Reply:
Updated.